### DISCUSSION TITLE
Are all nextjs templates free for commercial use?

### DISCUSSION THREAD

**Discussion by wenqi73** [vercel/community#3281] [Feature Question] [answered]
### Question

Hey, I need to build a company website. Like most company websites, it will support the following features:
- Product display
- Blog
- Payment

Are all nextjs templates(https://vercel.com/templates) free for commercial use? If not, what should I do?

**Comment by leerob** [comment, marked as answer]
Yes, unless specifically mentioned on the repository or in the license.
